pom.xml

```diff
@@ -99,6 +99,8 @@
     <useBeta>false</useBeta>
     <incrementals.url>https://repo.jenkins-ci.org/incrementals/</incrementals.url>
     <scmTag>HEAD</scmTag>
+    <!-- Where to put temporary files during test runs. -->
+    <java.io.tmpdir>${project.build.directory}/tmp</java.io.tmpdir>
```

Comment:
this is an **exceptionally bad** name for this property as you will have to pass it in maven with -Djava.io.tmpdir which will change the temp directory not only for surefire but all of maven and then when you clean if you have temp files open it will fail to work (temp files include the jar cache on some systems)

I would suggest surefireTempDir or something like that